string object_name
float32 score
int32 min_x
int32 max_x
int32 min_y
int32 max_y